packages/framework/src/Resources/views/Debug/Elasticsearch/template.html.twig

```diff
@@ -0,0 +1,139 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.data.requestsCount > 0 %}
+        {% if collector.data.totalRequestsTime * 1000 > 200 %}
```

Comment:
I'm just wondering why these numbers? (I would wonder for any number without explanation) Could you please comment in the code the reason?